download.py

```diff
@@ -91,51 +131,68 @@ def get_zone_links(czds_base_url):
 if not zone_links:
     exit(1)
 
+##############################################################################################################
+# Fourth Step: Set up AWS integration
+##############################################################################################################
 
+boto3.setup_default_session(**aws_session_args)
+aws_cloudwatch = boto3.client('cloudwatch', region_name=aws_region)
 
 ##############################################################################################################
-# Fourth Step: download zone files
+# Fifth Step: download zone files
 ##############################################################################################################
 
```

Comment:
For a top level function declaration, it is [good style](https://www.python.org/dev/peps/pep-0008/#blank-lines) to have two blank lines preceding it.